copy to do list src

juanmanuelbyc opened the pull request:
This is an exact copy of the source code of the project 'To Do List'. Here I expect my coding partner to write down all his suggestions about some changes that I could apply to this project in order to stick to the best practices.

Moathal reviewed (changes requested):
## Great job so far only few things to achieve perfection 👍 

- It will feel great if you add `keydown` `eventListener` when adding new task. you just need to check the key down in enter by adding `if(e.code = 'Enter')`

- it will look great if you `focusin` and `focusout` event listeners to change background color when click on the input field of task description.

- it would be better if you remove the indexes from tasks descriptions since it is nor required.

juanmanuelbyc commented:
Thanks for your feedback Moathal! I took into account your suggestions and made the following changes:

- Added  keydown eventListener when adding new task.

- Added focusin and focusout event listeners to change background color when click on the input field of task description.

- Removed the indexes from tasks descriptions since it is not required.

Moathal reviewed (approved):
Great everything is perfect you can submit 🥳 🍾
